src/protocol/OfflineMessage.php

```diff
@@ -46,4 +46,4 @@ protected function writeMagic(BinaryStream $out){
 	public function isValid() : bool{
 		return $this->magic === self::MAGIC;
 	}
-}
+}
```

Comment:
Unrelated change

Reply:
fwiw, by linux and git conventions, text files *should* have a trailing newline.